src/index.js

```diff
@@ -1,6 +1,7 @@
 var multiaddr = require('multiaddr')
 var getConfig = require('./config')
 var getRequestAPI = require('./request-api')
+var request = require('request')
```

Comment:
Not being able to confirm, but read somewhere that request gets polifilled in browserify. Nevertheless, there is https://www.npmjs.com/package/browser-request for that :)